core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -59,6 +59,9 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
     uint public expirationTimestamp;
     // Time that has to elapse for a withdrawal request to be considered passed, if no liquidations occur.
     uint public withdrawalLiveness;
+    // Time that has to elapse for the contract to be siphoned. This occures if a long period (>6 months) has
+    // occured post expiration and there is remaining unclaimed collateral in the contract.
```

Comment:
I don't think we can (or should) assume the siphonDelay is long or > 6 months.

Reply:
have removed this part of the comment